draft-ietf-tls-dtls13.md

```diff
@@ -582,6 +582,10 @@ tag and need no padding. However, some algorithms such as
 TLS_AES_128_CCM_8_SHA256 have a shorter authentication tag and may require padding
 for short inputs.
 
+Future cipher suites which are not based on AES or ChaCha20 MUST define
+their own record sequence number encryption in order to be used with
+DTLS.
```

Comment:
I can't decide whether this requirement merits adding this document as a reference for the ciphersuites registry.

Reply:
I would say yes, provided that it is additive.

Reply:
Agreed.

Reply:
What does this imply for the document itself? Do we need to capture it in the IANA consideration section or would we ask IANA to update the references for the TLS Cipher Suites registry at https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml?

Reply:
Typically we would have this document's IANA Considerations section request IANA to update the references for the registry to add this document as a reference.

Reply:
Done.